package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "channelape-sdk",
-	"version": "1.2.0",
+	"version": "1.2.1-develop.0",
```

Comment:
This should be 1.2.1 since your target is master with a hotfix. We will not be publishing an intermediary version.